orianna.py

```diff
@@ -365,7 +365,7 @@ def get_role_emoji(summoner):
             emoji_name = duo_codes[summoner.role]
         else:
             emoji_name = summoner.lane.lower()
-        emoji_code = emojis[emoji_name]
+        emoji_code = emojis.get(emoji_name, 'Emoji is not available')
```

Comment:
Using .get() prior to the code update, allowed the code to execute in games with Aphelios